commands/registry.go

```diff
@@ -80,12 +80,12 @@ func Registry() *Command {
 	cmdRunRegistryCreate.Example = `The following example creates a registry named ` + "`" + `example-registry` + "`" + ` in the NYC3 region: doctl registry create example-registry --region=nyc3`
 
 	getRegDesc := "Retrieves details about a private container registry, including its name and the endpoint used to access it."
-	cmdRegistryGet := CmdBuilder(cmd, RunRegistryGet, "get", "Retrieve details about a container registry",
+	cmdRegistryGet := CmdBuilder(cmd, RunRegistryGet, "get <registry-id>", "Retrieve details about a container registry",
```

Comment:
At the moment, only one container registry is supported per account. So this command is a bit of an outlier. There is no positional argument, just `doctl registry get`.

Reply:
oops.. you're right..
it actually works without registry id. i was specifying registry id as first parameter but seems it was actually ignored.....